interface/src/app/node/node.js

```diff
@@ -92,5 +92,223 @@ angular.module('adage.node', [
   }]
 )
 
+.directive('highRangeExp', ['$http', '$log', function($http, $log) {
+  return {
+    templateUrl: 'node/high_range_exp.tpl.html',
+    restrict: 'E',
+    scope: {
+      nodeId: '@',
+      inputTopNum: '@topExp'
+    },
+    link: function($scope) {
+      $scope.queryStatus = 'Connecting to the server ...';
```

Comment:
Are we using ``$scope`` instead of 'controllerAs' syntax here because this code is in the ``link`` function?